frontend/test/metabase/scenarios/dashboard/dashboard.cy.spec.js

```diff
@@ -67,7 +67,7 @@ describe("scenarios > dashboard", () => {
       .type("{selectall}Orders per year")
       .blur();
 
-    cy.wait("@updateDashboard");
+    cy.findByText("Edited a few seconds ago by you").should("be.visible");
```

Comment:
There seem to be other 2 API responses after waiting for `@updateDashboard` previously as seen in this screenshot.

![image](https://user-images.githubusercontent.com/1937582/220414836-a7b83fd4-80cc-4b4d-8c79-f4b7159c0e4a.png)

This fix makes sure we wait for those API calls to finish before clicking the info button.

Reply:
Does seem like a better thing to wait for haha. Thanks @WiNloSt